bin/pycbc_inspiral

```diff
@@ -92,10 +92,13 @@ parser.add_argument("--cluster-function", choices=["findchirp", "symmetric"],
 parser.add_argument("--cluster-window", type=float, default = -1,
                     help="Length of clustering window in seconds."
                     " Set to 0 to disable clustering.")
-parser.add_argument("--maximization-interval", type=float, default=0,
+parser.add_argument("--maximization-interval", nargs='+', type=float, default=0,
                     help="Maximize triggers over the template bank (ms)")
 parser.add_argument("--bank-veto-bank-file", type=str, help="FIXME: ADD")
-parser.add_argument("--chisq-snr-threshold", type=float, help="Minimum SNR to calculate the power chisq")
+parser.add_argument("--lat-snr-threshold", type=float)
+parser.add_argument("--lat-chisq-locations", type=str, nargs='+')
```

Comment:
Would need help strings ... But would prefer these options *not* to be in pycbc_inspiral.